Packs/FortiSandbox/Integrations/FortiSandbox/FortiSandbox.py

```diff
@@ -165,12 +164,7 @@ def upload_file_on_demand(session, base_url):
         file_sha256 = demisto.args().get('sha256')
 
     try:
-        shutil.copy(file_path, file_name)
-    except Exception:
-        raise Exception('Failed to prepare file for upload.')
-
```

Comment:
Just to make sure, no replacement is needed here?

Reply:
Waiting for customer to test the integration out after the code change. Stay tuned.

Reply:
Yeah this should be fine. The shutil.copy() command is replaced by just opening the file using the file path instead of copying the file into the container using shutil (This is a legacy way before getFilePath was added)